--- origsrc/gnormalize-0.63/Makefile	2008-06-10 20:03:39.000000000 -0500
+++ src/gnormalize-0.63/Makefile	2008-07-13 15:00:30.171875000 -0500
@@ -32,7 +32,7 @@
 	install -Dm 0644 icons/gnormalize-32.png "$(ICONDIR)/gnormalize.png"
 	install -Dm 0644 icons/gnormalize-48.png "$(LICONDIR)/gnormalize.png"
 	install -Dm 0644 icons/gnormalize-16.png "$(MICONDIR)/gnormalize.png"
-	ln -sf "$(ICONDIR)/gnormalize.png" "$(ICONDIR2)/gnormalize.png"
+	install -Dm 0644 icons/gnormalize-32.png "$(ICONDIR2)/gnormalize.png"
 
 postinstall:
 	update-menus
